proto/api.proto

```diff
@@ -134,6 +157,16 @@ message ResponseHeader {
   optional Transaction txn = 3;
 }
 
+// A TableRow represents a series of column values within a row in
+// a table. 
+message TableRow {
+  message ColumnValue {
+    optional bytes column_name = 1;
+    optional Value value = 2;
+  }
+  repeated ColumnValue columns = 1; 
```

Comment:
Including the column names in every `TableRow` is descriptive, but inefficient when returning multiple rows containing the same columns as in the case of a scan. I think it makes sense to return the column names and column values separately and perhaps not even have an explicit `TableRow` object.